limitador/src/storage/redis/redis_cached.rs

```diff
@@ -210,30 +226,56 @@ impl CachedRedisStorage {
         max_cached_counters: usize,
         ttl_cached_counters: Duration,
         ttl_ratio_cached_counters: u64,
+        response_timeout: Duration,
     ) -> Result<Self, RedisError> {
         let info = ConnectionInfo::from_str(redis_url)?;
-        let redis_conn_manager = ConnectionManager::new(
+        let redis_conn_manager = ConnectionManager::new_with_backoff_and_timeouts(
             redis::Client::open(info)
                 .expect("This couldn't fail in the past, yet now it did somehow!"),
+            2,
+            100,
+            6,
```

Comment:
Maybe it could be more informative if we extract this to default values in a `const` (?)

Reply:
They are... they actually are inlined from the redis crate, cause I can't reuse theirs ... not public.